code/go/0chain.net/validatorcore/storage/models_test.go

```diff
@@ -0,0 +1,879 @@
+package storage_test
+
+import (
+	"testing"
+
+	"0chain.net/core/common"
+	"0chain.net/core/config"
+	"0chain.net/core/logging"
+	"0chain.net/core/node"
+	"0chain.net/validatorcore/storage"
+
+	"github.com/0chain/gosdk/core/zcncrypto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+)
+
+func TestAttributes_String(t *testing.T) {
```

Comment:
not sure if this is the intended behavior, but `attributes.String()` returns `{}` when owner pays. I would think the check is done elsewhere?